test/integration_test/common_test.go

```diff
@@ -293,6 +297,12 @@ func setup() error {
 	if err == nil {
 		log.InfoD("Three cluster config mode has been activated for test: %t", externalTest)
 	}
+	// Fetch the object store provider.
+	provider = os.Getenv(providerEnv)
```

Comment:
Now, we have to change all jenkins piplines once this gets merged, right?

Reply:
No, I have tried to keep the pipeline changes backwards compatible. If the provider is not set, the default will be S3.